include/aws/auth/signing_config.h

```diff
@@ -137,17 +207,17 @@ struct aws_signing_config_aws {
      *
      */
 
-    /*
+    /**
      * AWS Credentials to sign with.
      */
     struct aws_credentials *credentials;
 
-    /*
+    /**
      * AWS credentials provider to fetch credentials from.
      */
     struct aws_credentials_provider *credentials_provider;
```

Comment:
Note that `aws_signing_config_aws.credentials` and `aws_signing_config_aws.credentials_provider` are mutually exclusive. Also note that `credentials_provider` can't be used with the streaming body signature types.

Reply:
there's already a comment detailing that above it